nestkernel/connection_manager.cpp

```diff
@@ -195,38 +201,40 @@ nest::ConnectionManager::set_synapse_status( index gid,
 void
 nest::ConnectionManager::delete_connections_()
 {
-  for ( tVSConnector::iterator it = connections_.begin();
-        it != connections_.end();
-        ++it )
+#ifdef _OPENMP
+#pragma omp parallel
   {
-    for ( tSConnector::nonempty_iterator iit = it->nonempty_begin();
-          iit != it->nonempty_end();
-          ++iit )
+#pragma omp for schedule( static, 1 )
```

Comment:
I think the explicit `for schedule( static, 1 )` here makes sense, since it will ensure a fixed assignment of threads to loop iterations. I would rather think we should introduce it in other places as well (might improve performance?).